tests/test_index_load.py

```diff
@@ -0,0 +1,15 @@
+try:
```

Comment:
Do you mind removing this? We would want this as a separate PR

Reply:
This was just a test, it shouldn't have been there.